enterprise_extensions/sampler.py

```diff
@@ -190,6 +190,11 @@ def draw_from_empirical_distr(self, x, iter, beta):
                 idx = self.pnames.index(self.empirical_distr[distr_idx].param_name)
                 q[idx] = self.empirical_distr[distr_idx].draw()
 
+                # TODO (Aaron): test this:
```

Comment:
Is this vestigial code that can be deleted?

Reply:
This segment has been tested and the line here has been removed.